src/pages/youngho/Login/Login.scss

```
@@ -0,0 +1,68 @@
+@import 'https://fonts.googleapis.com/css2?family=Lobster&display=swap';
+body {
+  background-color: #fafafa;
+}
+.container {
+  text-align: center;
+  border: 1px solid #e6e6e6;
+  background-color: white;
+  width: 600px;
+  height: 600px;
+  margin: 50px auto 0 auto;
+}
+.header {
+  font-family: 'Lobster', cursive;
+  font-size: 70px;
+  font-weight: 1px;
+  padding: 30px;
+}
```

```suggestion
body {
  background-color: #fafafa;
}

.container {
  text-align: center;
  border: 1px solid #e6e6e6;
  background-color: white;
  width: 600px;
  height: 600px;
  margin: 50px auto 0 auto;
}

.header {
  font-family: 'Lobster', cursive;
  font-size: 70px;
  font-weight: 1px;
  padding: 30px;
}
```
가독성을 위해 셀럭터 시작할 때는 한 줄 띄워주시기 바랍니다.

---

반영 완료했습니다!